src/main/ApplicationUtils.cpp

```diff
@@ -195,13 +125,13 @@ setupApp(Config& cfg, VirtualClock& clock, uint32_t startAtLedger,
         return nullptr;
     }
 
-    // With in-memory mode, ledger state is not yet ready during this setup step
-    app->getLedgerManager().loadLastKnownLedger(
-        /* restoreBucketlist */ false,
-        /* isLedgerStateReady */ !cfg.isInMemoryMode());
+    app->getLedgerManager().loadLastKnownLedger(/* restoreBucketlist */ false);
```

Comment:
shouldn't this go away now? IIRC the only reason we needed to load state here was to support in-memory mode (we call this function later in `start` anyway)